caddy_chatbot/src/caddy_core/services/retrieval_chain.py

```diff
@@ -99,13 +100,12 @@ def build_chain(CADDY_PROMPT):
         sorted=True,
     )
 
-    pipeline = DocumentCompressorPipeline(
-        transformers=[filter_ordered_by_retriever])
+    pipeline = DocumentCompressorPipeline(transformers=[filter_ordered_by_retriever])
     compression_retriever = ContextualCompressionRetriever(
         base_compressor=pipeline, base_retriever=lotr
     )
 
-    llm = BedrockChat(
+    llm = ChatBedrock(
```

Comment:
we might want to modularise this somehow so we're not hard coding bedrock use directly... but not sure best way of handling it.  For discusion tomorrow!

Reply:
I think pretty hard to no hardcode no as this is from langchain? Unless there's a nice langchain function that allows you to specify / switch between api resources easily